internal/desktop/desktop_platform_freebsd.go

```diff
@@ -0,0 +1,39 @@
+//go:build freebsd
+// +build freebsd
+
+package desktop
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/stashapp/stash/pkg/logger"
+)
+
+// isService checks if started by init, e.g. stash is a *nix systemd service
+func isService() bool {
+	return os.Getppid() == 1
+}
+
+func isServerDockerized() bool {
+	_, dockerEnvErr := os.Stat("/.dockerenv")
+	cgroups, _ := ioutil.ReadFile("/proc/self/cgroup")
+	if !os.IsNotExist(dockerEnvErr) || strings.Contains(string(cgroups), "docker") {
+		return true
+	}
+
+	return false
+}
```

Comment:
Those 2 functions are linux specific. 
Does freebsd use pid 1 for init or if an app is run a service ?
Does a docker container running in freebsd have the relevant files?
Both of those are used to check if stash is running in  desktop mode (not 100% sure about that) or not so either a correct way to detect that in freebsd is needed or just say that desktop mode is not supported and return a fixed value instead (probably `return true`?)

Reply:
To be honest, I just copied it from the linux-file, sorry. `isService` should return true to disable desktop and `isServerDockerized` should return false. There is no docker on FreeBSD as far as I knew. FreeBSD uses PID 1 for init, but you can run a service without having `PPID==1`. There is runit for example.

Reply:
I pushed a new and hopefully fixed version.

Reply:
And now I reverted the previous fix and applied it to the freebsd-file, not the linux-file. It's to late for me ;-)